docs/content/posts/2019-08-22-release-1.9.0.md

```diff
@@ -42,7 +42,7 @@ lists](https://flink.apache.org/community.html#mailing-lists) or
 feedback is very much appreciated!
 
 
-{% toc %}
+
```

Comment:
can you remove this line instead? There isn't really a reason to replace the toc with an empty line in the .md files (even if they dont get rendered)